keyboards/cannonkeys/satisfaction75/led.c

```diff
@@ -19,6 +19,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "led_custom.h"
 #include "satisfaction75.h"
 
+#ifdef QWIIC_MICRO_OLED_ENABLE
+#include "micro_oled.h"
```

Comment:
"no such file exists", in travis CI.

Reply:
only for the `tester` keymap, which has `QWIIC_ENABLE = no`... but that should result in `QWIIC_MICRO_OLED_ENABLE` not being defined: https://github.com/qmk/qmk_firmware/blob/master/drivers/qwiic/qwiic.mk